styles/Hero.module.css

```diff
@@ -0,0 +1,68 @@
+.container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    padding-bottom: var(--spacing-unit);
+}
+
+.hero_container {
+    margin-top: 319.5px;
+    display: flex;
+    flex-direction: column;
+    gap: 29.5px;
```

Comment:
Please use CSS vars from globals

Reply:
No variable or class in the CSS globals for flex column

Reply:
You can use a multiple from the single space

Reply:
fixed